### ISSUE TITLE
Funciones como resultado de reducciones

### ISSUE THREAD

**Issue by asanzo** [uqbar-project/function-laboratory#53]
Si quiero reducir `id even` no me deja:

![Selección_999(709)](https://user-images.githubusercontent.com/5421992/117979820-b3598c80-b309-11eb-9a72-4c907f6c6f3d.png)

Y si hago `map (+) [1,2,3,4]` estalla por los aires:

![Selección_999(708)](https://user-images.githubusercontent.com/5421992/117979858-be142180-b309-11eb-830c-92fc23957654.png)

**Comment by PalumboN** [comment]
JA! Sí, esto tiene la complicación de que todavía no devolvemos funciones como resultado de una reducción, esto nos trabó porque no sabríamos cómo mostrar esas funciones. Podrían ser `<una función>` _a la_ pdepreludat :thinking: 

Vos qué esperabas que devolvieran estas reducciones @asanzo ?

**Comment by asanzo** [comment]
y, acá tenés el bloque en la mano, así que si hacés que el compile devuelva una estructura con el bloque original además del resultado per sé, podés jugar a que 

`id even` reduzca a `even` y

`map (+) [1,2,3]` reduzca a `[(1+), (2+), (3+)]`

Pero puedo estar pidiendo algo demasiado zarpado.

**Comment by asanzo** [comment]
Sí, volviendo a este issue después de un tiempo creo que me quedo con el "<una función>" y con eso ya soy feliz  (especialmente si esa función tiene un agujero y está correspondientemente currificada).
